README.md

```diff
@@ -687,7 +687,20 @@ Multi-Model Endpoint can be used together with Pre/Post-Processing. Each model w
                 |--lib
                     |--external_module
                 |--inference.py
```

Comment:
How to provide model specific inference.py via SM SDK MME? Can you provide add notebook in SM examples?

Reply:
Updated the directory structure